LoopKitUI/Views/SettingDescription.swift

```diff
@@ -8,12 +8,26 @@
 
 import SwiftUI
 
+public enum LoopSetting: Int {
+    case glucoseTargetRange
+    case correctionRangeOverrides
+    case suspendThreshold
+    case basalRate
+    case deliveryLimits
+    case insulinModel
+    case carbRatio
+    case insulinSensitivity
+}
 
 public struct SettingDescription: View {
     var text: Text
+    var settingType: LoopSetting
```

Comment:
I'm not sure it makes sense to tie this view to a specific therapy setting. It would be clearer to take a `@ViewBuilder` closure for the help screen, then define the appropriate help screen within the setting's page.

Reply:
I believe the help screens are going to be reused between the flow and settings page, per conversations with design. I added a `@ViewBuilder` closure to the `SettingsDescription`, but moved the instructional page function to the enum.